src/screens/channels/ChannelListingsScreen/index.tsx.tsx

```diff
@@ -46,11 +56,33 @@ const Contents = ({ channelUrl }: { channelUrl: string }): ReactElement => {
 
   if (!fsChannel || (!isFetching && channelListings.length === 0)) {
     return (
-      <View style={styles.empty}>
-        <FormText font={'SB'} color={COLOR.black._300}>
-          No listings found
+      <SafeAreaView style={styles.empty}>
+        <Ionicons
+          name="alert-circle-outline"
+          size={44}
+          color={COLOR.black._300}
+          style={{ margin: 20 }}
+        />
+        <FormText
+          font={'B'}
+          size={18}
+          color={COLOR.black._900}
+          style={{ textAlign: 'center' }}
+        >
+          {t('Nft.ChannelListingNoListedNft')}
         </FormText>
-      </View>
+        <FormButton
+          figure="outline"
+          containerStyle={styles.buttonListNft}
+          textStyle={{ color: COLOR.black._300 }}
+          rightIcon="ios-chevron-forward"
+          onPress={(): void => {
+            // navigation.navigate(Routes.ListNft, { channelUrl })
```

Comment:
what is this?

Reply:
![스크린샷 2023-06-28 오후 2 45 59](https://github.com/palmapp-xyz/palm-mobile/assets/35752384/6aa8307b-8d58-4787-9643-0898566aab8e)

This button lists the nft when there is no nft listed.
Implementation of list nft view was not completed.
 I'll remove the button.